src/thunderbots/software/CMakeLists.txt

```diff
@@ -176,8 +181,22 @@ add_executable (radio_communication
 # Depend on exported targets (other packages) so that the messages in our thunderbots_msgs package are built first.
 # This way the message headers are always generated before they are used in compilation here.
 add_dependencies(radio_communication ${catkin_EXPORTED_TARGETS})
+
+##### LIBUSB #####
+pkg_search_module("LIBUSB_1" REQUIRED "libusb-1.0")
+include_directories(SYSTEM "${LIBUSB_1_INCLUDE_DIRS}")
+link_directories("${LIBUSB_1_LIBRARY_DIRS}")
+add_definitions("${LIBUSB_1_DEFINITIONS}")
+
+##### SIGC++  ######
+pkg_search_module("SIGC" REQUIRED "sigc++-2.0")
+include_directories(SYSTEM "${SIGC_INCLUDE_DIRS}")
+link_directories("${SIGC_LIBRARY_DIRS}")
+
 target_link_libraries(radio_communication ${catkin_LIBRARIES}
         ${G3LOG}
+        ${LIBUSB_1_LIBRARIES}
+	${SIGC_LIBRARIES}
```

Comment:
``` suggestion
	 ${SIGC_LIBRARIES}
```

Reply:
is this a space vs. tab issue?

Reply:
This might be GitHub displaying things weird actually. If you look at the raw version of the file I don't see this spacing issue. @henrymliu @garethellis0 can you confirm?

Reply:
Just took a look, it is a tab issue (I have vim setup to render tabs as `>---`):
![screenshot from 2019-01-27 21-18-36](https://user-images.githubusercontent.com/9075711/51815960-378ebf80-2279-11e9-81c5-63412ade0521.png)